functional-samples/tutorial.terminate-sw/test-extension/service-worker.js

```diff
@@ -0,0 +1,3 @@
+chrome.runtime.onMessage.addListener((message, sender, sendResponse) => {
```

Comment:
As mentioned in the doc comment, I think it'd be great if this would fail after the service worker gets suspended.